src/app/plugins/addroles.plugin.ts

```diff
@@ -69,12 +70,53 @@ export default class AddRolesPlugin extends Plugin {
       })
     );
 
+    // check for alumni or grad student role, and send a random welcome
+    filteredRoles.forEach((e) => {
+      const roleName = e.name.toLowerCase();
+      const alumniRole = Constants.Roles.Alumni.toLowerCase();
+      const gradStudRole = Constants.Roles.GradStudent.toLowerCase();
+
+      const alumniChannel = this.container.guildService.getChannel(Constants.Channels.Public.AlumniLounge) as TextChannel;
+      const gradStudChannel = this.container.guildService.getChannel(Constants.Channels.Public.GradCafe) as TextChannel;
+
+      if (roleName === alumniRole) {
+        alumniChannel.send(this._getRandomWelcome(member.toString(), alumniRole));
+      }
+
+      if (roleName === gradStudRole) {
+        gradStudChannel.send(this._getRandomWelcome(member.toString(), gradStudRole));
+      } 
+    });
```

Comment:
I might want to change this pattern, but I will take a look after the other changes are made.